packages/components/src/theme/createTheme.ts

```diff
@@ -0,0 +1,63 @@
+import type { ThemeDefinition } from 'vuetify';
+import type {
+  Themes, Mode, ContrastColor, Theme,
+} from '@atb-as/theme';
+
+const VUETIFY_THEME_KEYS = [
+  'background',
+  'surface',
+  'surface-bright',
+  'surface-light',
+  'surface-variant',
+  'primary',
+  'info',
+  'error',
+  'success',
+  'warning',
+];
+
+// Mapping from Vuetify variable to design token
+const VuetifyMap = (theme: Theme): Record<string, ContrastColor> => ({
+  background: theme.color.background.neutral[0],
+  surface: theme.color.background.neutral[0],
+  'surface-bright': theme.color.background.neutral[1],
+  'surface-light': theme.color.background.neutral[3],
+  'surface-variant': theme.color.background.accent[2],
+  primary: theme.color.interactive[3].default,
+  'primary-darken-1': theme.color.interactive[3].hover,
```

Comment:
Here is a source of confusion. If the intent is to darken the color, we have no guarantee that the color used for :hover is what we want. Darken means darken, not hover. I'd rather see a computed solution (css filter, js manipulation …) for this. Obviously OMS doesn't have darken/lighten variants which Vuetify has a plethora of.

Reply:
You're right, but at this time OMS is missing colors. This was a way to "make it look like" RTM before this change, but maybe we should prefer hardcoding this until we add for example "brand-primary" to the OMS design system?

Reply:
Hard-coding is preferred in this case, yes :-)